cli/command/run/cmd.go

```diff
@@ -69,25 +69,8 @@ func Run(cmd *cobra.Command, cli *command.SauceCtlCli, args []string) (int, erro
 		return 1, err
 	}
 
-	// Merge env from CLI args and job config. CLI args take precedence.
-	for k, v := range env {
-		cfg.Env[k] = v
-	}
-
-	if testTimeout != 0 {
-		cfg.Timeout = testTimeout
-	}
-	if cfg.Timeout == 0 {
-		cfg.Timeout = defaultTimeout
-	}
-
-	if cfg.Sauce.Region == "" {
-		cfg.Sauce.Region = defaultRegion
-	}
-
-	if region != "" {
-		cfg.Sauce.Region = region
-	}
+	mergeArgs(&cfg)
+	expandEnvMetadata(&cfg)
```

Comment:
These seem to be very specific operations for the config and could maybe live in the [config package](https://github.com/saucelabs/saucectl/tree/master/cli/config)

Reply:
Actually, let me add that as a function of metadata itself.